### ISSUE TITLE
Strange response time and lack of spotter functionality for Assetto Corsa v1.12 (64-bit)

### ISSUE THREAD

**Issue by Stewie410** [mrbelowski/CrewChiefV4#65]
In AC v1.11 at the start of the race CrewCheif would tell me the track temps and such before leaving the car-setup menu.  Currently, the application waits until 30 seconds (or so) into the race before talking about track temps--which isn't really an issue, but as far as I know this is not intended behavior.  Likewise, the spotter utility seems to be totally nonexistent with AC's latest build.

**Comment by TheIronWolfModding** [comment]
Hi Stewie,

You may want to join and post in AC section of http://thecrewchief.org/ for higher visibility.

Vytautas

**Comment by mrbelowski** [comment]
So you're not getting any 'car left' / 'car right' calls with 1.12? They added some data to one of the shared memory blocks but I thought it was at the end. I'll investigate later and update here

**Comment by mrbelowski** [comment]
Just to check... The crew chief python app hasn't been disabled by the update (in the assetto UI) has it?

**Comment by Stewie410** [comment]
Nope--that was the first thing I checked was whether or not the app hadn't been disabled (in Settings>General).  Checked the in-game UI and saw that the UI element had been disabled--I reenabled the in-game UI element, restarted AC (and CC) and did some test races to see what all happened.  The spotter seems to be running, though things still seem out of sync initially, then it will correct itself.

Actually, I the last test race I did ended with me spinning out and quickly closing the game--and now both all "still there" clips continue to repeat while the game is closed and CC remains running; not a big deal, and it could be a one-off exception.

The "track temps" and other pre-green messages still seem to be delayed as well; not entirely sure what all was in the new update that could have changed that, though.

~So, re-enabling the UI element seems to have brought the spotter back, though there still seems to be a bit of lag in the playing of the audio files at the start of the race, including the spotter function.  Attached are two logs of instances of "lag" in CC's audio clips at the start of a race--the first taking place at Brands Hatch GP with no modded-cars being used, and the latter at the Fort Wayne Oval modded track.  Both locations produced similar results, with the slight variant of the race and Brands Hatch GP's (which I exited shortly after spinning out) which resulted in a indefinite loop of audio playback--as shown in the log file.

[BrandsHatchGP_OddBehavior_CCv4.5Log.txt](https://github.com/mrbelowski/CrewChiefV4/files/784119/BrandsHatchGP_OddBehavior_CCv4.5Log.txt)
[FortWayneOval_OddBehavior_CCv4.5Log.txt](https://github.com/mrbelowski/CrewChiefV4/files/784118/FortWayneOval_OddBehavior_CCv4.5Log.txt)
